consensus/fast-sync/src/fast_sync.rs

```diff
@@ -0,0 +1,217 @@
+use crate::{hash_of_hashes, BlockId, HashOfHashes};
+
+#[allow(unused_imports)]
+use hex_literal::hex;
+
+use std::{
+    cmp,
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+};
+use tower::Service;
+
```

Comment:
Same here with the imports:

```suggestion
use std::{
    cmp,
    future::Future,
    pin::Pin,
    task::{Context, Poll},
};

#[allow(unused_imports)]
use hex_literal::hex;
use tower::Service;

use crate::{hash_of_hashes, BlockId, HashOfHashes};
```